File: Length_Weight_Volume_Unit_Converter.java
```java
import java.util.Scanner;

public class UnitConverter {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Choose type of conversion \n 1. Length \n 2. Weight \n 3. Volume");
        int choice = scanner.nextInt();
        switch (choice) {
            case 1:
                System.out.println("Enter value in meters");
                double meters = scanner.nextDouble();
                System.out.println("In feet : " + meters * 3.28084);
                System.out.println("In inches : " + meters * 39.3701);
                break;
            case 2:
                System.out.println("Enter value in kilograms");
                double kilograms = scanner.nextDouble();
                System.out.println("In pounds : " + kilograms * 2.20462);
                System.out.println("In ounces : " + kilograms * 35.274);
                break;
            case 3:
                System.out.println("Enter value in liters");
                double liters = scanner.nextDouble();
                System.out.println("In gallons : " + liters * 0.264172);
                System.out.println("In cubic feet : " + liters * 0.0353147);
                break;
            default:
                System.out.println("Invalid choice");
        }
        scanner.close();
    }
}
```

File: Length_Weight_Volume_Unit_Converter.py
def weight_conversion():
    print("1. Grams to Kilograms")
    print("2. Kilograms to Pounds")
    print("3. Pounds to Ounces")
    choice = int(input("Enter your choice: "))
    if choice == 1:
        g = float(input("Enter weight in grams: "))
        kg = g / 1000
        print("Weight in kilograms: ", kg)
    elif choice == 2:
        kg = float(input("Enter weight in kilograms: "))
        lb = kg * 2.20462
        print("Weight in pounds: ", lb)
    elif choice == 3:
        lb = float(input("Enter weight in pounds: "))
        oz = lb * 16
        print("Weight in ounces: ", oz)